charts/datadog/README.md

```diff
@@ -803,6 +803,8 @@ helm install <RELEASE_NAME> \
 | datadog.otelCollector.config | string | `nil` | OTel collector configuration |
 | datadog.otelCollector.enabled | bool | `false` | Enable the OTel Collector |
 | datadog.otelCollector.ports | list | `[{"containerPort":"4317","name":"otel-grpc"},{"containerPort":"4318","name":"otel-http"}]` | Ports that OTel Collector is listening |
+| datadog.otelCollector.rbac.create | bool | `true` | If true, create RBAC resources for OTel Collector |
```

Comment:
I think it's a bit unclear from name / description that it's for `k8sattributes` processor. The field that creates the rbac in the otel helm charts is named [kubernetesAttributes](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/opentelemetry-operator-0.79.0/charts/opentelemetry-collector/values.yaml#L50) (although that one also adds the processor to pipelines). I think either changing the name or description to make this clearer would be good